To: vim_dev@googlegroups.com
Subject: Patch 7.3.324
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.324 (after 7.3.237)
Problem:    Completion for ":compiler" shows color scheme names.
Solution:   Fix the directory name. (James Vega)
Files:	    src/ex_getln.c


*** ../vim-7.3.323/src/ex_getln.c	2011-08-17 16:25:43.000000000 +0200
--- src/ex_getln.c	2011-09-30 16:22:14.000000000 +0200
***************
*** 4546,4552 ****
      }
      if (xp->xp_context == EXPAND_COMPILER)
      {
! 	char *directories[] = {"colors", NULL};
  	return ExpandRTDir(pat, num_file, file, directories);
      }
      if (xp->xp_context == EXPAND_OWNSYNTAX)
--- 4546,4552 ----
      }
      if (xp->xp_context == EXPAND_COMPILER)
      {
! 	char *directories[] = {"compiler", NULL};
  	return ExpandRTDir(pat, num_file, file, directories);
      }
      if (xp->xp_context == EXPAND_OWNSYNTAX)
*** ../vim-7.3.323/src/version.c	2011-09-30 14:44:49.000000000 +0200
--- src/version.c	2011-09-30 16:23:15.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     324,
  /**/


-- 
hundred-and-one symptoms of being an internet addict:
31. You code your homework in HTML and give your instructor the URL.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
